README.md

```diff
@@ -139,52 +134,112 @@ Previous simulation platforms that have operated on similar datasets typically p
 
 To run the above benchmarks on your machine, see instructions in the [examples](#examples) section.
 
-## Quick installation
+## Installation
+
+### Docker Image
+
+We provide a pre-built docker container for habitat-api and habitat-sim, refer to [habitat-docker-setup](https://github.com/facebookresearch/habitat-api#docker-setup).
+
+### From Source
+
+We highly recommend installing an [Anaconda](https://www.anaconda.com/distribution/#download-section) environment (note: python>=3.6 is required). Once you have Anaconda installed, here are the instructions.
```

Comment:
```suggestion
We highly recommend installing an [miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/distribution/#download-section) environment (note: python>=3.6 is required). Once you have Anaconda installed, here are the instructions.
```

Add pointer to miniconda -- it is much lighter weight to install and we don't need everything in pre-packaged anaconda.